### PR TITLE
Buffs mag-gun recharges speed

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]
Makes mag guns rechargers charge faster

:cl: optional name here
tweak: charging
/:cl:

[why]
Kev told me so, even did the pinging

### CODE REVIEW

File: modular_citadel/code/modules/projectiles/guns/ballistic/magweapon_energy.dm [hunk 1]
```diff
@@ -74,10 +74,12 @@
 /obj/item/stock_parts/cell/magrifle_e
 	name = "magrifle power supply"
 	maxcharge = 14400
```
Comment (Useroth) [inline modular_citadel/code/modules/projectiles/guns/ballistic/magweapon_energy.dm:76]: Why not reduce the maxcharge and e_cost of ammunition instead? If you divided both by 5,
```suggestion
	maxcharge = 2880
```
you would end up making it charge slower, but not all that significantly, than most e-guns (but you get 2 magazines worth of shooting per charge).

Or if you want them to charge almost exactly fast as regular e-guns,
```suggestion
	maxcharge = 1440
```

Comment (Trilbyspaceclone) [reply modular_citadel/code/modules/projectiles/guns/ballistic/magweapon_energy.dm:76]: Doing this way allows it to basically be EMP proof still

**Review by deathride58 — APPROVED** [review]
Can be merged with approval from one other maintainer. Personally, I'm torn on whether or not Useroth's idea would be better or if this is fine

**Comment by davidchan13** [comment]
Bit off topic but would it be possible to add a means to see how much charge is left in the mag gun? Either a percentage like power cells on examine or lights on the mag-gun that turn off as it nears zero, similiar to energy guns?

**Comment by Useroth** [comment]
https://github.com/Citadel-Station-13/Citadel-Station-13/pull/8320 @davidchan13

**Pull request merged**